src/setup/compatibility.c

```diff
@@ -27,7 +27,7 @@
 int vanilla_savegame_limit = 1;
 int vanilla_demo_limit = 1;
 
-void CompatibilitySettings(void)
```

Comment:
Does changing `(void)` to `()` work? Might be cleaner.

Reply:
No, it looks like to be a bug in Emscripten cdecl functions. Opened https://github.com/kripken/emscripten/issues/7011

I'd wait for their reply about cdecl implementation before going forward with this PR.

Reply:
Emscripten folks gave a good explanation how WebAssembly requires the function signatures to match and @turol also pointed out that we shouldn't depend on this undefined behavior so I'm now also convinced this PR is in fact correct.

Reply:
Thanks for following up on this.